pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/Annotateable.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.ast;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface Annotateable extends JavaNode {
+    List<ASTAnnotation> getDeclaredAnnotations();
```

Comment:
You should add the documentation for these methods here and remove it on the implementation (it will be inherited)